tfjs-layers/src/layers/nlp/tokenizers_test.ts

```diff
@@ -19,17 +19,38 @@
  * Unit Tests for Tokenizer Layers.
  */
 
-import { Tensor1D, tensor1d } from '@tensorflow/tfjs-core';
+import { Tensor1D, serialization, tensor1d } from '@tensorflow/tfjs-core';
 
-import { WhiteSpaceTokenizer } from './tokenizers';
-import { expectTensorsClose } from '../../../src/utils/test_utils';
+import { Tokenizer } from './tokenizers';
+import { expectTensorsClose } from '../../utils/test_utils';
 
-describe('White Space Tokenizer', () => {
-  const tokenizer = new WhiteSpaceTokenizer();
+class SimpleTokenizer extends Tokenizer {
+  /** @nocollapse */
+  static className = 'SimpleTokenizer';
+
+  tokenize(inputs: Tensor1D): Tensor1D[] {
+    const stringInputs = inputs.dataSync() as unknown as string[];
+    return stringInputs.map(input => tensor1d(input.split(' ')));
+  }
+
+  override detokenize(inputs: Tensor1D[]): Tensor1D {
+    const stringInputs = inputs.map(
+      input => input.dataSync() as unknown as string[]);
+    return tensor1d(stringInputs.map(str => str.join(' ')));
+  }
+}
+serialization.registerClass(SimpleTokenizer);
```

Comment:
We're not saving or loading models that use this, so we don't need to register it (since it's just a test class).

```suggestion
```

Reply:
done